assets/js/googlesitekit/datastore/user/user-input-settings.js

```diff
@@ -159,6 +177,18 @@ const baseResolvers = {
 };
 
 const baseSelectors = {
+	/**
+	 * Determines whether the user input settings are being saved or not.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {boolean} TRUE if the user input settings are being saved, otherwise FALSE.
+	 */
+	isSavingUserInputSettings( state ) {
+		return !! state?.isSavingInputSettings;
+	},
```

Comment:
I saw your comment on the PR, but can you clarify what the problem was with `isFetchingSaveUserInputSettings`? Since the above calls `fetchSaveUserInputSettings`, why can't we rely on that?

Reply:
@felixarntz the `isFetchingSaveUserInputSettings` selector works only when we pass the appropriate argument (an array of user input settings without scopes) into it which we can't get in the user input components. So, it has never worked as expected.